launcher/package.json

```diff
@@ -0,0 +1,69 @@
+{
+  "name": "che-code-launcher",
+  "publisher": "eclipse-che",
+  "version": "0.0.1",
+  "private": false,
+  "description": "Configure and launch VS Code",
+  "main": "out/entrypoint.js",
+  "bin": "out/entrypoint.js",
+  "publishConfig": {
+    "access": "public"
+  },
+  "files": [
+    "out",
+    "src"
+  ],
+  "scripts": {
+    "prepare": "yarn run clean && yarn run build",
+    "clean": "rimraf lib",
+    "build": "yarn run format && yarn run compile && yarn run lint && yarn run test",
+    "compile": "tsc --project .",
+    "format": "prettier --check '{src,tests}/**/*.ts' package.json",
+    "format:fix": "prettier --write '{src,tests}/**/*.ts' package.json",
+    "lint": "eslint --cache=true --no-error-on-unmatched-pattern=true '{src,tests}/(!model|**)/*.ts'",
+    "lint:fix": "eslint --fix --cache=true --no-error-on-unmatched-pattern=true \"{src,tests}/(!model|**)/*.{ts,tsx}\"",
+    "test": "jest --forceExit",
+    "watch": "tsc -w",
+    "publish:next": "yarn publish  --registry=https://registry.npmjs.org/ --no-git-tag-version --new-version 0.0.1-\"$(date +%s)\""
```

Comment:
Are we really going to publish the launcher?

Reply:
It was copypasted from another project, thanks!